lib/Backend/GlobOptFields.cpp

```diff
@@ -382,7 +382,10 @@ GlobOpt::KillAllFields(BVSparse<JitArenaAllocator> * bv)
     bv->ClearAll();
     if (this->IsLoopPrePass())
     {
-        this->rootLoopPrePass->allFieldsKilled = true;
+        for (Loop * loop = this->rootLoopPrePass; loop != nullptr; loop = loop->parent)
+        {
+            loop->allFieldsKilled = true;
```

Comment:
We have the same issue when setting loop->fieldKilled.  Can you please fix this as well?